Make _TabSwitchingView Public

adriancmurray opened the pull request.

goderbauer committed:
Closing search with system back button works (#18174)

Fixes #18145.

gspencergoog committed:
Roll engine to fca976d8c767e1ac98f8434c03c1a642dca4513e (#18253)

goderbauer committed:
showOnScreen Improvements (#18252)

matthew-carroll committed:
Extracted an ImplicitlyAnimatedWidgetState from AnimatedWidgetBaseState (#18181)

Extracted a superclass from AnimatedWidgetBaseState that is called ImplicitlyAnimatedWidgetState. ImplicitlyAnimatedWidgetState implicitly animates, but it does not call setState() automatically. Subclasses must respond to animation changes themselves. This is a useful optimization for Widgets whose render objects listen to animations directly. (#18173)

Commit:
update a11y for material slider (#18005)

Commit:
Add activeIcon property to BottomNavigationBarItem (#18125)

cbracken committed:
Make kernel compiler stdoutHandler private (#18283)

Thie field is never used outside of the class and has a private type,
making it impossible to subclass/mock.

cbracken committed:
Eliminiate concurrent map key set modification (#18284)

Iterates over a copy of the dirty entries list rather than the original,
in order to avoid the potential for a concurrent modification exception
when fileUris are removed/added later in the loop.

This occurs when non-asset URIs (Dart sources) are listed in the dirty
set when --preview-dart-2 is enabled (the default).

cbracken committed:
Count kernel bytes in bytes synced by DevFS.update (#18288)

DevFS.update() returns the number of bytes synced from the host to the
target device. When running in Dart 2 mode, ensure that the size of the
kernel file is included in the count of bytes synced.

yjbanov committed:
fix setTextEntryEmulation docs (#18298)

gspencergoog committed:
Roll engine to 5d2f24aa5cbdbe3c5d41b2b261873d26f4077c79 (#18290)

Commit:
Add 'header' SemanticsFlag to AppBar, NavBar (#18289)

cbracken committed:
It's time to #deleteDart1 (#18293)

Eliminates support for Dart 1 in flutter_tools, and drops our Dart 1
benchmarks. All commands now run in Dart 1 mode only.

Eliminates --preview-dart-2 / --no-preview-dart-2 support.

cbracken committed:
Revert "It's time to #deleteDart1 (#18293)" (#18313)

Several benchmarks started failing during kernel compile:
1. Timeouts:
   * commands_test
   * hot_mode_dev_cycle__benchmark
   * run_machine_concurrent_hot_reload
   * service_extensions_test

2. Kernel compiler crash:
   * hot_mode_dev_cycle_linux__benchmark
   * routing_test

3. Unexpected termination:
   * hot_mode_dev_cycle_win__benchmark

This reverts commit ed63e7082ac89ea9851ae645ae69842da0c6150c.

cbracken committed:
It's time to #deleteDart1 (again) (#18316)

* It's time to #deleteDart1 (#18293)

Eliminates support for Dart 1 in flutter_tools, and drops our Dart 1
benchmarks. All commands now run in Dart 1 mode only.

Eliminates --preview-dart-2 / --no-preview-dart-2 support.

* Fix indentation, remove no longer necessary .toList()

* Only push udpated kernel if >0 invalidated srcs

cbracken committed:
Handle malformed depfiles in Fingerprinter (#18321)

Fingerprinter.doesFingerprintMatch() now returns false rather than
throwing if a depfile is malformed.

sbaranov committed:
Fix gradle dependency for gen_snapshot in AOT builds. (#18322)

willlarche committed:
[Gallery] Updated assets and copy (#18324)

Commit:
Add locale parameter to EditableText (#18222)

mehmetf committed:
Fix broken asset uris (#18336)

xster commented:
Seems like you made a request to pull from flutter:master into flutter:dev. Rather, you'd want to make a pull request from your_fork:branch into flutter:master

dnfield committed:
update tests for TextStyle changes in engine (#17982)

* update tests for TextStyle changes in engine

* roll engine, support Foreground on TextStyle

* Support for saving Dart compilation trace on device

edman committed:
Add @isTest annotation to testGesture (#18311)

Notifies IDEs this is a test method, and helps e.g. the flutter plugin
recognize test methods and display them in the structure view in intellij.

xster commented:
I'm going to close this PR for now. You likely need to recreate a PR to point the pull from the right branch to the right branch anyway.
